Inferences/FunctionDefinitionRewriting.cpp

```diff
@@ -64,7 +64,8 @@ Clause* performRewriting(
 
   Applicator appl(subst.ptr());
 
-  if (helper && !helper->isPremiseRedundant(rwClause,rwLit,rwTerm,tgtTermS,eqLHS,&appl)) {
+  Ordering::Result temp;
```

Comment:
Perhaps it would be worth adding an overload for the "we don't care about the ordering result" case.